src/Clock.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  date: Date;
+};
+
+export class Clock extends React.Component<Props, State> {
+  timerId = 0;
+
+  state = {
+    date: new Date(),
+  };
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+    }, 1000);
+  }
```

Comment:
You missed a task requirement:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/178909951-4ee98813-f5d6-4dd6-8738-9b25b133534f.png">

```suggestion
  componentDidMount() {
    this.timerId = window.setInterval(() => {
      this.setState({ date: new Date() });
    }, 1000);
  }
```